bokehjs/test/integration/regressions.ts

```diff
@@ -574,6 +574,23 @@ describe("Bug", () => {
     })
   })
 
+  describe("in issue #10498", () => {
+    it("rebuild GridBox when rows or cols properties are modified", async () => {
```

Comment:
This needs to read as a complete sentence. Currently it's:
```
Bug in issue #10498 rebuild GridBox when rows or cols properties are modified"
```
Preferably it should read:
```
Bug in issue #10498 prevents GridBox from rebuilding when rows or cols properties are modified
```